.zuul.yaml

```diff
@@ -124,23 +103,46 @@
     roles:
       - zuul: github.com/openstack-k8s-operators/ci-framework
     required-projects: *required_projects
-    irrelevant-files: *irrelevant_files
-    files:
-      - roles/telemetry_graphing/.*
-      - .zuul.yaml
-      - ci/vars-graphing-test.yml
-      - ci/run_graphing_test.yml
+
 
 - project:
     name: infrawatch/feature-verification-tests
     github-check:
       jobs:
-        - feature-verification-tests-noop
+        - feature-verification-tests-noop:
+            files: *irrelevant_files
         - openstack-k8s-operators-content-provider:
             override-checkout: main
             irrelevant-files: *irrelevant_files
         - functional-tests-on-osp18:
-            voting: true
```

Comment:
Was the voting removal intentional or a typo?

Reply:
It could be either.

voting is true by default, so removing it should not change behaviour